test/test_autograd.py

```diff
@@ -7793,6 +7793,20 @@ def jvp(ctx, x_tangent, y_tangent):
             with self.assertRaisesRegex(RuntimeError, "You should return None at that position instead"):
                 FuncWrong.apply(x_dual, y)
 
+        # returns non-tensor
+        class Func(torch.autograd.Function):
+            @staticmethod
+            def forward(ctx, x):
+                return x.clone(), object()
```

Comment:
```suggestion
                return x.clone(), object(), x.clone()
```
Just to make sure there isn't any off indexing happening!